packages/kinvey-nativescript-sdk/src/bundle.ts

```diff
@@ -29,7 +29,6 @@ export {
   Log,
   Metadata,
   Query,
-  User,
```

Comment:
Why is `User` removed here?

Reply:
It was removed on this line because the `User` module is now exported by the [line](https://github.com/Kinvey/js-sdk/pull/305/files/b1f5ab460d683c7c6160e6b8d018098d2750a5c2#diff-10d68474809f3791aaa0c9fbeca55222R75). The `User` module is now extended by the NativeScript package.